istio-control/istio-discovery/templates/configmap-envoy.yaml

```diff
@@ -113,19 +109,46 @@ data:
           tls_context:
             require_client_certificate: true
             common_tls_context:
+              alpn_protocols:
+              - h2
+
               validation_context:
```

Comment:
if sds enabled, the root cert is also fetched through sds (so here in validation_context needs to be sdsconfig), could be updated in later PR.

Reply:
It could be - but to minimize risks ( and take advantage of cert rotation and multi-cluster) we can also load it from the mounted file. 

This reduces the load ( less generated config ) and work ( since SDS doesn't currently support all the merging and custom roots )